### PR TITLE
Don't pre-populate everywhere

### PR DESCRIPTION
**Pull request by tpendragon**
Reform pre-population is meant as a way to build up filler nested objects for display in an HTML form, not to transform values. This should make it easier to follow Reform documentation.

### CODE REVIEW

File: spec/queries/find_members_with_relationship_spec.rb [hunk 1]
```diff
@@ -7,16 +7,16 @@
 
   describe "#find_members_with_relationship" do
     it "returns members with `loaded_relationship` loaded" do
```
Comment (escowles) [inline spec/queries/find_members_with_relationship_spec.rb]: How are these changes related to the `prepopulate` change?

File: spec/resources/coins/coin_feature_spec.rb [hunk 2]
```diff
@@ -127,7 +127,7 @@
     let(:artist) { FactoryBot.create_for_repository(:numismatic_artist) }
 
     before do
-      visit solr_document_path coin
+      visit solr_document_path id: coin.id.to_s
```
Comment (escowles) [inline spec/resources/coins/coin_feature_spec.rb]: Why is this change needed?  Wouldn't it route to the same page either way?

File: spec/services/manifest_builder/license_builder_spec.rb [hunk 3]
```diff
@@ -20,21 +20,5 @@
         expect(manifest["license"]).to eq "https://creativecommons.org/licenses/by-nc/4.0/"
       end
     end
```
Comment (escowles) [inline spec/services/manifest_builder/license_builder_spec.rb]: How are these changes related to the `prepopulate` change?

File: spec/derivative_services/external_metadata_derivative_service_spec.rb [hunk 4]
```diff
@@ -57,6 +57,6 @@
     parent = query_service.find_by(id: parent_resource.id)
     expect(parent.title).to eq ["China census data by county, 2000-2010"]
     expect(parent.visibility).to eq ["open"]
-    expect(event_generator).to have_received(:record_updated).twice
+    expect(event_generator).to have_received(:record_updated).exactly(3).times
```
Comment (escowles) [inline spec/derivative_services/external_metadata_derivative_service_spec.rb]: Why did this change?

Comment (tpendragon) [reply spec/derivative_services/external_metadata_derivative_service_spec.rb]: I actually dug around this, and I have no idea. It seemed like it was firing everything at correct times, I'm not sure how it only fired twice before.

File: spec/services/manifest_builder/license_builder_spec.rb (hunk 3 above)
Comment (tpendragon) [reply spec/services/manifest_builder/license_builder_spec.rb]: I have no idea how this worked before! It seems like it was a bug. If there's a license field, why would it just be empty if you save a value?

File: spec/resources/coins/coin_feature_spec.rb (hunk 2 above)
Comment (tpendragon) [reply spec/resources/coins/coin_feature_spec.rb]: Oh, yes, I can get rid of this. The problem was that it was only persisting to postgres before, but the change set was being marked as changed as part of creating the record, so the change_set_persister was being fired which would re-save it and it'd end up in solr.

File: spec/queries/find_members_with_relationship_spec.rb (hunk 1 above)
Comment (tpendragon) [reply spec/queries/find_members_with_relationship_spec.rb]: Oh! This one was interesting - it wasn't pre-populating before. It's actually impossible to save a change_set with two genres because it's marked as a single field in the ChangeSet. Now that it happens on initialize, I had to use a field which was marked as multiple to get two things to show up.

File: spec/derivative_services/external_metadata_derivative_service_spec.rb (hunk 4 above)
Comment (tpendragon) [reply spec/derivative_services/external_metadata_derivative_service_spec.rb]: I just dug through this again. There are 3 events:

1. The change_set is saved for the first time.
2. Saving the change_set for the first time fixes the visibility of the FileSet that's attached.
3. The metadata is extracted, firing the updated for the record again.
